hoot-core-test/src/test/cpp/hoot/core/io/ImplicitTagRulesSqliteWriterTest.cpp

```diff
@@ -0,0 +1,246 @@
+/*
+ * This file is part of Hootenanny.
+ *
+ * Hootenanny is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * --------------------------------------------------------------------
+ *
+ * The following copyright notices are generated automatically. If you
+ * have a new notice to add, please use the format:
+ * " * @copyright Copyright ..."
+ * This will properly maintain the copyright information. DigitalGlobe
+ * copyrights will be updated automatically.
+ *
+ * @copyright Copyright (C) 2017 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
Looks like the copyright stuff needs to be run on the new files.

Reply:
That copyright script was pretty f'd up when I ran it.  I ended up running it three times to get all the files updated, and some of them it wouldn't pick up, so I edited them by hand.  The multiple lines are being added by that script...I guess I can just hand edit them all.